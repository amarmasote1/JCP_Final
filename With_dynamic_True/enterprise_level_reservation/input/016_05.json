[
  {
    "storeID": "1180",
    "itemID": "17345670016",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 20
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  }
]