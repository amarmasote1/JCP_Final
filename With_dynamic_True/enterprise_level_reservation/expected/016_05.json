{"status":200,"message":"Success","body":[{"itemID":"17345670016","uom":"EACH","storeID":"1180","supplyData":{"onhandsupply":20},"demandData":{"reserveddemand":0}}]}